src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services1.yml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
To be reverted?

Reply:
This is a fix because a newline is always added before `services:` even if there is no `parameters`.